cmd/node_agent/na/onprem_test.go

```diff
@@ -61,3 +82,56 @@ func TestGetServiceIdentity(t *testing.T) {
 		}
 	}
 }
+
+func TestGetDialOptions2(t *testing.T) {
+	testCases := map[string]struct {
+		config      *Config
+		expectedErr string
+	}{
+		"Good cert": {
+			config: &Config{
+				CertChainFile:  "testdata/cert.pem",
```

Comment:
Can you give better names for these files? Or consider reusing the existing ones if possible?

Reply:
Renamed to TestGetTLSCredentials

Reply:
Sorry, I meant testdata/cert.pem. We have a bunch of test files already, we need to give them better names.

Reply:
Renamed files names